dropwizard-e2e/src/test/java/com/example/badlog/BadLogTest.java

```diff
@@ -28,18 +28,15 @@
     private static PrintStream oldOut = System.out;
     private static PrintStream oldErr = System.err;
 
-    @Rule
-    public TemporaryFolder folder = new TemporaryFolder();
```

Comment:
Does this mean the temporary file allocated in the test will need to be deleted?

Reply:
Those files will be created in the `/tmp` directory on UNIX-systems and the OS will automatically take care of cleaning them. I agree that explicit removing is better, but we use temporary files sparingly in the test suite, so that shouldn't be an issue.

Reply:
JUnit 5.4.0 re-introduced the [`TempDirectory` extension](https://junit.org/junit5/docs/5.4.0/user-guide/index.html#writing-tests-built-in-extensions-TempDirectory).